app/scenes/more_dms/index.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+
+import navigationSceneConnect from '../navigationSceneConnect';
+
+import {goBack} from 'app/actions/navigation';
+import {makeDirectChannel} from 'app/actions/views/more_dms';
+import {getProfiles, searchProfiles} from 'service/actions/users';
+import {getMyPreferences, getTheme} from 'service/selectors/entities/preferences';
+import {searchedProfiles} from 'service/selectors/entities/users';
+
+import MoreDirectMessages from './more_dms';
+
+function mapStateToProps(state, ownProps) {
+    const {getProfiles: requestStatus, searchProfiles: searchRequest} = state.requests.users;
+
+    function getUsers() {
```

Comment:
yes it will, I'm not a fan but then it's the only way I made it work, lets merge this and keep and eye on performance